manifests/f/FreeDV/FreeDV/1.8.4/FreeDV.FreeDV.locale.en-US.yaml

```
@@ -0,0 +1,32 @@
+# Created using wingetcreate 1.2.8.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: FreeDV.FreeDV
+PackageVersion: 1.8.4
+PackageLocale: en-US
+Publisher: FreeDV

```

```suggestion
Publisher: CMake
```
Must match Control Panel / Apps&Features